File: app/src/main/java/com/weathercock/cleanarchitecture/data/entity/TemperatureEntity.java
package com.weathercock.cleanarchitecture.data.entity;

import io.realm.RealmObject;

/**
 * Temperature Entity
 * <p>
 * Created by tamura_k on 2016/06/03.
 */
public class TemperatureEntity extends RealmObject {

    //最高気温
    private TemperatureUnitEntity max;
    //最低気温
    private TemperatureUnitEntity min;

    public TemperatureUnitEntity getMax() {
        return max;
    }

    public void setMax(TemperatureUnitEntity max) {
        this.max = max;
    }

    public TemperatureUnitEntity getMin() {
        return min;
    }

    public void setMin(TemperatureUnitEntity min) {
        this.min = min;
    }

}

File: app/src/main/java/com/kazakago/cleanarchitecture/data/entity/weather/TemperatureEntity.kt
package com.kazakago.cleanarchitecture.data.entity.weather

import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

@Entity
class TemperatureEntity(
        @PrimaryKey(autoGenerate = true)
        val temperatureId: Int,

        //最高気温
        val max: TemperatureUnitEntity?,
        //最低気温
        val min: TemperatureUnitEntity?
)